manager/controllers/utils.go

```diff
@@ -183,12 +184,19 @@ func ListCommon(tx *gorm.DB, c *gin.Context, path string, opts ListOpts) (*gorm.
 		TotalItems: total,
 	}
 
+	var tagQ string
```

Comment:
Please, separate it to `fucntion extractTagsQueryString(c) string` and you can remove //nolint: lll